ocb3/src/lib.rs

```diff
@@ -55,7 +56,9 @@ pub type Nonce<NonceSize> = Array<u8, NonceSize>;
 /// OCB3 tag
 pub type Tag<TagSize> = Array<u8, TagSize>;
 
-pub(crate) type Block = Array<u8, U16>;
+type BlockSize = U16;
+pub(crate) type Block = Array<u8, BlockSize>;
+type DoubleBlock = Array<u8, <BlockSize as Mul<U2>>::Output>;
```

Comment:
Alternatively you can use `Add` instead of `Mul<U2>` (where `Add` implicitly takes `Self` as a default) e.g. https://github.com/RustCrypto/AEADs/blob/a09f3d2/aes-siv/src/siv.rs#L101

Reply:
It's an internal type, I think it's better to be explicit about the intent?
Relying on the implicit value of Add makes it harder to read imho.

Reply:
went with `typenum::Prod`